app/controllers/api/feedbacks_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::FeedbacksController < Api::BaseController
+
+  skip_before_filter :authenticate_user
+
+  def create
+    feedback = Feedback.new feedback_params(params)
+    if feedback.save
+      render json: { message: 'Received' }, status: :created
+    else
+      unprocessable_response feedback
+    end
+  end
+
+  private
+
+  def feedback_params(params)
```

Comment:
`params` is automagically passed through, so this can just be `feedback_params`.